gatling/user-files/utils/delete-users.js

```diff
@@ -0,0 +1,33 @@
+const
+  rp = require('request-promise'),
+  host = process.argv[2] || 'localhost';
+
+const doit = async (id) => {
+  for (let it = 1; it < 100; ++it, ++id) {  
```

Comment:
Same comments as in the `create-users.js` file, meaning:
* the `id` function argument should not be modified
* the magic number used both here and in the runner function should be replaced with a constant